app/uk/gov/hmrc/gform/sharedmodel/email/LocalisedEmailTemplateId.scala

```diff
@@ -16,19 +16,29 @@
 
 package uk.gov.hmrc.gform.sharedmodel.email
 
-import play.api.libs.json.{ JsError, JsObject, JsString, JsSuccess, Reads }
+import play.api.libs.json._
+import uk.gov.hmrc.gform.sharedmodel.formtemplate.OFormatWithTemplateReadFallback
 import uk.gov.hmrc.gform.sharedmodel.{ EmailVerifierService, LangADT, LocalisedString }
 import uk.gov.hmrc.gform.sharedmodel.notifier.NotifierTemplateId
 
-case class LocalisedEmailTemplateId(emailTemplateId: String, maybeEmailTemplateIdCy: Option[String]) {
-  def toDigitalContact = EmailVerifierService.digitalContact(
-    EmailTemplateId(emailTemplateId)
-  )
+case class LocalisedEmailTemplateId(emailTemplateId: String, emailTemplateIdCy: Option[String]) {
+  def toDigitalContact =
+    EmailVerifierService.digitalContact(
+      EmailTemplateId(emailTemplateId),
+      emailTemplateIdCy.map(EmailTemplateId.apply)
+    )
+
   def toNotify =
     EmailVerifierService.notify(
       NotifierTemplateId(emailTemplateId),
-      maybeEmailTemplateIdCy.map(NotifierTemplateId.apply)
+      emailTemplateIdCy.map(NotifierTemplateId.apply)
     )
+
+  def getEmailTemplateId(lang: LangADT) =
```

Comment:
This is duplication of the same method in `case class DigitalContact`. Remove this one and use the other one instead